### ISSUE TITLE
Opinión: Sustituir Bootstrap 5 por Bulma

### ISSUE THREAD

**Issue by adriaanbd** [adriaanbd/pythonpanamaweb#4]
Que opinan de sustituir Boootstrap 5 por Bulma? Esta idea me la hizo Emmanuel no hace mucho y recién vi la página y documentación de Bulma: https://bulma.io/

Alguién aquí ha tenido la experiencia de desarrollar en Bulma vs Bootstrap 5? Any thoughts?

**Comment by fahenao** [comment]
Todos mis proyectos usan Bootstrap. Estoy dispuesto a intentar con Bulma.

Tiene sus puntos a favor. Por lo visto es un proyecto pequeño con buenos acabados y fácil de customizar.

El downside es que no tiene JS. Habra que escribir cualquier funcionalidad para el front end.
Como es una comunidad pequeña, los updates demoran en salir. 

Acá les dejo un resumen comparativo entre Bulma y Bootstrap:
https://programadorwebvalencia.com/es-mejor-bootstrap-o-bulma/

**Comment by adriaanbd** [comment]
Muchas gracias @fahenao eso fue enlightening. Si pude encontrar que hay bulma react components en React por lo que se pudiera implementar asi. Sin embargo, creo que mucho mas gente sabe Bootstrap que Bulma. Si el objetivo es que más gente pueda contribuir creo que Bootstrap cumple con ese objetivo. Bulma es como ese shiny new object!

**Comment by fahenao** [comment]
Agree.. Bootstrap es mas popular y tiene más documentación. Aunque aprender react no estaría mal 👀

**Comment by softd3v** [comment]
> Todos mis proyectos usan Bootstrap. Estoy dispuesto a intentar con Bulma.
> 
> Tiene sus puntos a favor. Por lo visto es un proyecto pequeño con buenos acabados y fácil de customizar.
> 
> El downside es que no tiene JS. Habra que escribir cualquier funcionalidad para el front end.
> Como es una comunidad pequeña, los updates demoran en salir.
> 
> Acá les dejo un resumen comparativo entre Bulma y Bootstrap:
> https://programadorwebvalencia.com/es-mejor-bootstrap-o-bulma/

Es correcto este framework no tiene JS incluido por lo que sirve para maquetar sitios con un fin mas basico.  Por el momento por lo que se quiere manejar puede ser que nos funcione.

**Comment by softd3v** [comment]
> Agree.. Bootstrap es mas popular y tiene más documentación. Aunque aprender react no estaría mal 👀

Es un buen punto.
